### PR TITLE
Add function added

### PR DESCRIPTION
**Pull request by math2018avan**

### CODE REVIEW

File: MyConsoleApp/MyConsoleApp.Test/MyProgramTest.cs [hunk 1]
```diff
@@ -7,11 +7,11 @@ public class MyProgramTest
         [Fact]
         public void VerifyGet()
```
Comment (MathavanN) [inline MyConsoleApp/MyConsoleApp.Test/MyProgramTest.cs]: Change the test case name
also add more test cases

**Review by MathavanN — CHANGES_REQUESTED** [review]
Please make the necessary changes

File: MyConsoleApp/MyConsoleApp.Test/MyProgramTest.cs (hunk 1 above)
Comment (math2018avan) [reply MyConsoleApp/MyConsoleApp.Test/MyProgramTest.cs]: made the necessary changes

**Review by MathavanN — APPROVED** [review]
LGTM

**Pull request merged**